README.md

```diff
@@ -1,10 +1,21 @@
-# Peer review
+# Red Hat peer review guide for technical documentation
 
 ## About
 
-The purpose of this project is to create guidelines for peer reviewing Red Hat product documentation.
+This guide documents best practices for peer reviewing Red Hat technical documentation. The guide also discusses tools to use when reviewing documentation, so that writers and reviewers can choose a method that best fits their workflow. We have also included a condensed checklist for reviewers to use as a quick reference.
```

Comment:
```suggestion
This guide documents best practices for peer reviewing Red Hat technical documentation. The guide also discusses tools to use when reviewing documentation, so that writers and reviewers can choose a method that best fits their workflow. We have also included a checklist for reviewers to use as a quick reference.
```